Source/WebKit/WebProcess/WebPage/wc/WCUpateInfo.h

```diff
@@ -308,6 +318,8 @@ struct WCUpateInfo {
     template <class Decoder>
     static WARN_UNUSED_RETURN bool decode(Decoder& decoder, WCUpateInfo& result)
     {
+        if (!decoder.decode(result.remoteContextHostedIdentifier))
```

Comment:
It would be nice if all this could be converted to WCUpdateInfo.serialization.in